sdk/maps/maps-search/package.json

```diff
@@ -0,0 +1,155 @@
+    "build:nodebrowser": "dev-tool run bundle",
+    "build:samples": "dev-tool samples publish --force",
+    "build:test": "tsc -p . && dev-tool run bundle",
+    "build:types": "downlevel-dts types/latest types/3.1",
+    "build": "tsc -p . && npm run build:nodebrowser && api-extractor run --local && npm run build:types",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* temp types *.tgz *.log coverage coverage-browser",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "autorest --typescript ./swagger/README.md",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 'test/**/*.spec.ts'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json README.md src test --ext .ts,.javascript,.js --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json README.md src test --ext .ts,.javascript,.js",
+    "pack": "npm pack 2>&1",
+    "ci": "npm run build && npm run check-format && npm run lint",
+    "test:browser": "npm run clean && npm run build:test && npm run integration-test:browser",
+    "test:node": "npm run clean && tsc -p . && npm run integration-test:node",
+    "test": "npm run clean && tsc -p . && npm run unit-test:node && dev-tool run bundle && npm run unit-test:browser && npm run integration-test",
+    "unit-test:browser": "npm run integration-test:browser",
+    "unit-test:node": "npm run integration-test:node",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/maps-common/src/",
+    "dist-esm/maps-search/src/",
+    "types/latest/",
+    "types/3.1/",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": "github:Azure/azure-sdk-for-js",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "keywords": [
+    "azure",
+    "cloud",
+    "typescript",
+    "maps",
+    "search"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/maps/maps-search/README.md",
+  "sideEffects": false,
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "dependencies": {
+    "@azure/abort-controller": "^1.0.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-client": "^1.0.0",
+    "@azure/core-http": "^2.0.0",
```

Comment:
do you need core-http?

Reply:
No, I don't. Thanks for the reminder. Also, I removed `@azure/core-paging` and `@azure/maps-common` since they are not needed. Fixed in 58afa842533e82f264f5c5f11ccd2fcd038d351b